app/services/bibid_updater.rb

```diff
@@ -21,29 +21,32 @@ def update
   private
 
     def change_set_persister
-      @change_set_persister ||= ChangeSetPersister.new(metadata_adapter: Valkyrie.config.metadata_adapter,
-                                                       storage_adapter: Valkyrie.config.storage_adapter)
+      @change_set_persister ||= ScannedResourcesController.change_set_persister
```

Comment:
You like this better? I think it's fine in a temporary service like this but I don't know that I'd want it sprinkled about.

Reply:
The previous one was actually wrong. If we started doing this for real I think `ChangeSetPersister.default` would be good.

Reply:
👍🏻 something like that would be good.

Reply:
Created https://github.com/pulibrary/figgy/issues/4629